L1Trigger/Phase2L1ParticleFlow/python/L1NNTauProducer_cff.py

```diff
@@ -1,7 +1,31 @@
 import FWCore.ParameterSet.Config as cms
 
-from L1Trigger.Phase2L1ParticleFlow.L1NNTauProducer_cfi import *
+from L1Trigger.Phase2L1ParticleFlow.l1ctLayer1_cff import l1ctLayer1Barrel,l1ctLayer1HGCal,l1ctLayer1
+
+#from L1Trigger.Phase2L1ParticleFlow.L1NNTauProducer_cfi import *
+
+#L1NNTauProducerPuppi = L1NNTauProducer.clone(
+#                                NNFileName      = cms.string("L1Trigger/Phase2L1ParticleFlow/data/tau_3layer_puppi.pb")
+#                                )
+
+
+L1NNTauProducerPuppi = cms.EDProducer("L1NNTauProducer",
```

Comment:
```suggestion
l1NNTauProducerPuppi = cms.EDProducer("L1NNTauProducer",
```
module labels should be lowercase